pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/OverloadShedder.java

```diff
@@ -121,8 +121,8 @@
                 }).filter(e ->
                         localData.getBundles().contains(e.getLeft())
                 ).sorted((e1, e2) -> {
-                    // Sort by throughput in reverse order
-                    return Double.compare(e2.getRight(), e1.getRight());
+                    // Sort by throughput in ascending order
```

Comment:
Are you changing "pick the biggest N bundles" to "pick the smallest N bundles" to unload?
It's better you could come up with more solid reasons to do so.
Using the smallest still won't avoid repeated unloading completely.